java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -60,6 +65,17 @@ class DevToolsService : Service() {
                 storageClass = extras.getSerializable("STORAGE_CLASS") as Class<StorageService>
             }
             initialize()
+
+            proxy = storageService?.getDevToolsProxy()
+            proxy?.registerBindingContextProxyMessageCallback(object : IStorageServiceCallback.Stub() {
```

Comment:
When the DevToolsService is destroyed (or the Binder is destroyed), this code should be a good citizen and un-register as well.